Source/Tests/Trakt.NET.Objects.Get.Tests/Shows/Json/Reader/ShowCastAndCrewObjectJsonReader/ShowCastAndCrewObjectJsonReader_Json_String_Tests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using FluentAssertions;
+using System.Linq;
+using System.Threading.Tasks;
+using Trakt.NET.Tests.Utility.Traits;
+using TraktNet.Objects.Get.Shows.Json.Reader;
+using Xunit;
+
+namespace TraktNet.Objects.Get.Tests.Shows.Json.Reader
+{
+    [Category("Objects.Get.Shows.JsonReader")]
+    public partial class ShowCastAndCrewObjectJsonReader_Tests
+    {
+        [Fact]
+        public async Task Test_ShowAliasObjectJsonReader_ReadObject_From_Json_String_Complete()
+        {
+            var jsonReader = new ShowCastAndCrewObjectJsonReader();
+
+            var traktShowCastAndCrew = await jsonReader.ReadObjectAsync(JSON_COMPLETE);
+
+            traktShowCastAndCrew.Cast.First().Characters.First().Should().Be("Daenerys Targaryen");
```

Comment:
Please don't use LINQ (`First()`).
Here you can write `traktShowCastAndCrew.Cast[0].Characters...`

Reply:
![image](https://user-images.githubusercontent.com/27026029/94939438-735c3c80-04d2-11eb-9399-1e43fac8785f.png)

Am I missing something here?

Reply:
Ah, okay. Yes, doesn't work without minimum LINQ. :-D

```csharp
ITraktShowCastMember[] cast = traktShowCastAndCrew.Cast.ToArray();
cast[0].Characters.First().Should().Be("Daenerys Targaryen");
```

Reply:
So should I convert the collections and then assert them or leave it the way it is?

Reply:
You can do it like in this file:
https://github.com/henrikfroehling/Trakt.NET/blob/a198696d11542c5235bcbee2dc42e1052774bda1/Source/Tests/Trakt.NET.Objects.Get.Tests/People/Credits/Json/Reader/PersonMovieCreditsObjectJsonReader/PersonMovieCreditsObjectJsonReader_Json_String_Tests.cs#L24